src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+
+class ByeCommand extends Command {
+    @Override
+    public void execute(TaskList tasks, boolean silent) {
```

Comment:
Consider using more descriptive names for your boolean variables and methods to enhance code readability. I suggest renaming 'silent' to 'isSilent' to make its purpose clearer.